### PR TITLE
Cancel coworker invite

### PR DESCRIPTION
**Pull request by pixelfact**

### CODE REVIEW

File: app/settings/billing/[id]/coworkers/page.tsx [hunk 1]
```diff
@@ -89,18 +89,14 @@ function CoworkersPage() {
...
-          } else if (canRemove) {
+          }
+          if (canRemove) {
             return "delete";
-          } else if (!isDisabled) {
-            return "disable";
-          } else {
-            return "enable";
           }
+          return "delete";
```
Comment (alexbeno) [inline app/settings/billing/[id]/coworkers/page.tsx:99]: check this default action with backend

File: components/features/stacks/payments-flow/request-payments-stacks/features/views/select-rewards/select-rewards.tsx [hunk 2]
```diff
@@ -119,7 +123,10 @@ export function SelectRewards({
             />
           </div>
           <div className="absolute bottom-0 left-0 w-full bg-greyscale-900">
-            <AmountCounter total={totalAmountSelectedRewards} isCompany={user?.billingProfileType === "COMPANY"} />
+            <AmountCounter
+              total={totalAmountSelectedRewards}
+              isCompany={user?.billingProfileType === BillingProfilesTypes.type.Company}
```
Comment (alexbeno) [inline components/features/stacks/payments-flow/request-payments-stacks/features/views/select-rewards/select-rewards.tsx:128]: maybe you can have isCompany const, you use this condition to time in this file

File: components/features/stacks/payments-flow/request-payments-stacks/features/views/select-rewards/select-rewards.tsx [hunk 3]
```diff
@@ -39,7 +40,10 @@ export function SelectRewards({
       includedRewards,
       excludedRewards,
     });
-    if (user?.billingProfileType === "INDIVIDUAL" || (user?.billingProfileType === "COMPANY" && isMandateAccepted)) {
+    if (
+      user?.billingProfileType === BillingProfilesTypes.type.Individual ||
+      (user?.billingProfileType === BillingProfilesTypes.type.Company && isMandateAccepted)
```
Comment (haydencleary) [inline components/features/stacks/payments-flow/request-payments-stacks/features/views/select-rewards/select-rewards.tsx:45]: Could use `isIndividual` and `isCompany` here

**Review by alexbeno — APPROVED** [review]
Comment will be done in another PR

**Pull request merged**